.github/workflows/build-and-test.yml

```diff
@@ -118,63 +118,14 @@ jobs:
           jq -r '.targets[].name' | \
           parallel -k -j 4 --retries 3 cargo run --example {} --all-features --release
 
-      - name: Stop sccache
-        uses: './.github/actions/rust/sccache/stop-sccache'
-        with:
-          os: ${{matrix.os}}
-
-  build-and-test-stardust:
```

Comment:
We should also remove the redundant steps in `format.yml` and `clippy.yml`, now that `identity_stardust` is part of the main workspace.